### ISSUE TITLE
Fix docs for rounded corners (example code not matching screenshot)

### ISSUE THREAD

**Issue by bcutter** [custom-cards/bar-card#156]
https://github.com/custom-cards/bar-card#200-default-layout-requires-card-mod shows rounded corners, while the example code is not working.

According to https://github.com/custom-cards/bar-card/issues/137 one needs to add

```
style: |-
  bar-card-currentbar, bar-card-backgroundbar {
    border-radius: 4px;
  }
```

Please FIX the docs so other users don't run into this issue.

Details: https://github.com/custom-cards/bar-card/issues/137#issuecomment-1344656202

**Comment by ButterBetzi** [comment]
Hi,
Im trying to round my corners in the same way you did in the screenshot and with this docu pr. But its not rounding for me. Could you confirm if this still works for you today?

**Comment by bcutter** [comment]
> Hi, Im trying to round my corners in the same way you did in the screenshot and with this docu pr. But its not rounding for me. Could you confirm if this still works for you today?

Yes working still quite well for me - but my HA Core Frontend is version 20230411.1 - maybe meanwhile few changes have been introduced.

**Comment by ButterBetzi** [comment]
Thanks for the quick answer!
Ah, ok, that might be it. Im running the latest version and cannot get those dawn corners rounded :D
But this seems to be my problem as I saw that this repo isnt maintained since 2020

**Comment by bcutter** [comment]
Well, not officially unmaintained (no notice), but yeah 10 open PRs, nobody seems to take care...

@jackjohnsonuk @Gluwc do you have write access to merge PRs and can ideally elaborate on the current status of this repo please?
